scripts/migration.py

```diff
@@ -0,0 +1,106 @@
+# Copyright (c) Microsoft Corporation.
```

Comment:
Can we rename the script to merge-databases.py as it will be part of the wider upgrade flow